lib/web/websocket/connection.js

```diff
@@ -91,7 +91,7 @@ function establishWebSocketConnection (url, protocols, client, ws, onEstablish,
   // 9. Let permessageDeflate be a user-agent defined
   //    "permessage-deflate" extension header value.
   // https://github.com/mozilla/gecko-dev/blob/ce78234f5e653a5d3916813ff990f053510227bc/netwerk/protocol/websocket/WebSocketChannel.cpp#L2673
-  const permessageDeflate = options.node?.['client-extensions'] ?? ''
+  const permessageDeflate = 'permessage-deflate'
```

Comment:
the server can simply ignore this; we should always advertise that we support it though.